packages/client/lib/types.ts

```diff
@@ -1,7 +1,7 @@
 import { EventEmitter } from 'events'
 import type { Multiaddr } from 'multiaddr'
 import type { Block, BlockHeader } from '@ethereumjs/block'
-import type Connection from '../../../node_modules/libp2p-interfaces/dist/src/connection/connection'
+import type Connection = require('../../../node_modules/libp2p-interfaces/dist/src/connection/connection')
```

Comment:
Why?

Reply:
` import type Connection from '../../../node_modules/libp2p-interfaces/dist/src/connection/connection' `

does not work because there is no "default" export from `connection`, and 
`import * as Connection from`
does not work because it `Connection` in that case is an object not a type.